Firmware/ControladorRobo/ControladorRobo.ino

```diff
@@ -93,14 +93,10 @@ void loop()
     basespeed *= 1.001;
 }
 
-///////////////////////////////////////////////////////////funcoes//////////////////////////////////////////////////////////////////////////////
-
-
-// interrupcao para TIMER1, periodo de amostragem para controle de velocidade
+// TIMER1 interruption, sample time for velocity control
 ISR(TIMER1_COMPA_vect)
 {
-    // Calculo das velocidades dos motores
-
+    // calculus of motor velocity
```

Comment:
Calculation of motor speed ?
Calculus seems wrong here.
And "velocity" implies a vector.